x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/search/BasicQueryClient.java

```diff
@@ -137,6 +140,9 @@ public void fetchHits(Iterable<List<HitReference>> refs, ActionListener<List<Lis
                 // the default size is 10 so be sure to change it
                 // NB:this is different from mget
                 .size(idQuery.ids().size());
+            if (fetchFields != null) {
+                fetchFields.stream().forEach(f -> builder.fetchField(f));
```

Comment:
`forEach` exists directly on `List` - no need to create the stream.
Also use the method reference : `fetchFields.forEach(builder::fetchField)`

Reply:
Indeed. Thanks.